### ISSUE TITLE
hi, run cli segmentation fault in now os 

### ISSUE THREAD

**Issue by qdore** [mdreizin/chrome-bookmarks-alfred-workflow#32]
./bin/cli
[1]    18242 segmentation fault  ./bin/cli

![image](https://user-images.githubusercontent.com/6831829/33307653-4f8e5e72-d452-11e7-985c-5606a675e6d6.png)

**Comment by qdore** [comment]
rt

**Comment by qdore** [comment]
Knock...Knock

**Comment by oalders** [comment]
Also seeing this on High Sierra:

```
[2018-01-19 13:41:47][input.scriptfilter] Queuing argument 'p'
[2018-01-19 13:41:48][input.scriptfilter] Script with argument '(null)' finished
[2018-01-19 13:41:48][ERROR: input.scriptfilter] Code 11: Exited with error code but no message
```

**Comment by oalders** [comment]
Ah, this should probably be closed as a dupe of #29

**Comment by mdreizin** [comment]
@qdore @oalders Hello guys, I'm really sorry for delayed response. It was easy to fix, but I didn't have free time. Please update to the latest version from https://github.com/mdreizin/chrome-bookmarks-alfred-workflow/releases/tag/0.1.9
